internal/lang/funcs/sensitive_test.go

```diff
@@ -180,3 +180,78 @@ func TestNonsensitive(t *testing.T) {
 		})
 	}
 }
+
+func TestIssensitive(t *testing.T) {
+	tests := []struct {
+		Input     cty.Value
+		Sensitive bool
+		WantErr   string
+	}{
+		{
+			cty.NumberIntVal(1).Mark(marks.Sensitive),
+			true,
+			``,
+		},
+		{
+			cty.NumberIntVal(1),
+			false,
+			``,
+		},
+		{
+			cty.DynamicVal.Mark(marks.Sensitive),
+			true,
+			``,
+		},
+		{
+			cty.UnknownVal(cty.String).Mark(marks.Sensitive),
+			true,
+			``,
+		},
+		{
+			cty.NullVal(cty.EmptyObject).Mark(marks.Sensitive),
+			true,
+			``,
+		},
+		{
+			cty.NullVal(cty.String),
+			false,
+			``,
+		},
+
+		// Unknown values may become sensitive once they are known, so we
```

Comment:
nit: I don't think this comment makes much sense. Unknown and null values can be marked as sensitive directly so don't need to worry about what happens to them when they become known.